SimPEG/electromagnetics/static/resistivity/simulation_2d.py

```diff
@@ -578,7 +579,7 @@ class Problem2D_N(BaseDCSimulation_2D):
     fieldsPair_fwd = Fields_N
 
     def __init__(self, mesh, **kwargs):
-        BaseDCSimulation_2D.__init__(self, mesh, **kwargs)
+        BaseDCSimulation2D.__init__(self, mesh, **kwargs)
```

Comment:
Just a thought, not blocking this: Would `super()` here be better? Just in case some things get switched around in the future, we are only ever changing things in one place.

Reply:
I agree, `super()` here would be super. However there are MANY places in the code that need this change, so I'm not as concerned about it just yet.

Reply:
Absolutely. :)